### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/zeek/commit/594a75aa0c34b8b9e5f8162efa821966e7d035b7#commitcomment-87673290

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/zeek/commit/8886aa33edf77be20795c9b5ae14a35d7344f0ca

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ jobs:
         env:
           RELEASE_VERSION: ${{ steps.version.outputs.RELEASE_VERSION }}
         run: |
-          # Translate the Github reference into a tag name.
+          # Translate the GitHub reference into a tag name.
```
Comment (jsoref) [inline .github/workflows/docker.yml:64]: Brand

File: NEWS [hunk 2]
```diff
@@ -1072,7 +1072,7 @@ New Functionality
   An example of a set with composite index is ``set[string, count, count]``.
 
 - Sumstats now allows manual epochs. If an ``epoch`` interval of 0 is specified,
-  epochs will have to be manually ended by callis ``SumStats::next_epoch``. This
+  epochs will have to be manually ended by calling ``SumStats::next_epoch``. This
```
Comment (jsoref) [inline NEWS:1075]: ?

File: ci/alpine/Dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 FROM alpine:latest
 
-# A version field to invalide Cirrus's build cache when needed, as suggested in
+# A version field to invalidate Cirrus's build cache when needed, as suggested in
```
Comment (jsoref) [inline ci/alpine/Dockerfile:3]: ?

File: scripts/base/frameworks/broker/store.zeek [hunk 4]
```diff
@@ -157,7 +157,7 @@ export {
 	## Returns: the result of the query.
 	global get: function(h: opaque of Broker::Store, k: any): QueryResult;
 
-	## Insert a key-value pair in to the store, but only if the key does not
+	## Insert a key-value pair into the store, but only if the key does not
```
Comment (jsoref) [inline scripts/base/frameworks/broker/store.zeek:160]: `into` has tended to be the most controversial change i've offered of late. happy to drop it or anything else

File: scripts/base/frameworks/openflow/consts.zeek [hunk 5]
```diff
@@ -61,7 +61,7 @@ export {
 	# IEEE 802.1X
 	const ETH_EAP_OVER_LAN = 0x888E;
 	# IEEE 802.1ad & IEEE 802.1aq
-	const ETH_PROVIDER_BRIDING = 0x88A8;
+	const ETH_PROVIDER_BRIDGING = 0x88A8;
```
Comment (jsoref) [inline scripts/base/frameworks/openflow/consts.zeek:64]: dunno if this is a public symbol or tied to something else.

File: scripts/base/frameworks/openflow/types.zeek [hunk 6]
```diff
@@ -102,7 +102,7 @@ export {
 		hard_timeout: count &default=0;
 		## Priority level of flow entry.
 		priority: count &default=0;
-		## For OFPFC_DELETE* commands, require matching entried to include
+		## For OFPFC_DELETE* commands, require matching entry to include
```
Comment (jsoref) [inline scripts/base/frameworks/openflow/types.zeek:105]: or entries?

File: scripts/base/frameworks/sumstats/cluster.zeek [hunk 7]
```diff
@@ -227,7 +227,7 @@ global stats_keys: table[string] of set[Key] &read_expire=1min
 		return 0secs;
 		};
 
-# This variable is maintained by manager nodes to track how many "dones" they
+# This variable is maintained by manager nodes to track how many "nodes" they
```
Comment (jsoref) [inline scripts/base/frameworks/sumstats/cluster.zeek:230]: ?

File: scripts/base/protocols/smb/const-dos-error.zeek [hunk 8]
```diff
@@ -109,7 +109,7 @@ redef SMB::statuses += {
 	[0x070a0001] = [$id="printeralreadyexists", $desc="The printer already exists."],
 	[0x070b0001] = [$id="invalidprintercommand", $desc="The printer command is invalid."],
 	[0x070c0001] = [$id="invaliddatatype", $desc="The specified datatype is invalid."],
-	[0x070d0001] = [$id="invalidenvironment", $desc="The Environment specified is invalid."],
+	[0x070d0001] = [$id="invalidatenvironment", $desc="The Environment specified is invalid."],
```
Comment (jsoref) [inline scripts/base/protocols/smb/const-dos-error.zeek:112]: This is probably wrong.

File: scripts/policy/protocols/ssl/weak-keys.zeek [hunk 9]
```diff
@@ -30,7 +30,7 @@ export {
 	## Warn if the DH key length is smaller than the certificate key length. This is
 	## potentially unsafe because it gives a wrong impression of safety due to the
 	## certificate key length. However, it is very common and cannot be avoided in some
-	## settings (e.g. with old jave clients).
+	## settings (e.g. with old java clients).
```
Comment (jsoref) [inline scripts/policy/protocols/ssl/weak-keys.zeek:33]: ?

File: src/CompHash.h [hunk 10]
```diff
@@ -45,7 +45,7 @@ class CompositeHash
 
 	// Compute the size of the composite key.  If v is non-nil then
 	// the value is computed for the particular list of values.
-	// Returns 0 if the key has an indeterminant size (if v not given),
+	// Returns 0 if the key has an indeterminate size (if v not given),
```
Comment (jsoref) [inline src/CompHash.h:48]: ?

File: src/Hash.h [hunk 11]
```diff
@@ -360,7 +360,7 @@ class HashKey
 protected:
 	char* CopyKey(const char* key, size_t size) const;
 
-	// Payload setters for types stored directoly in the key_u union. These
+	// Payload setters for types stored directory in the key_u union. These
```
Comment (jsoref) [inline src/Hash.h:363]: directly?

File: src/analyzer/Analyzer.cc [hunk 12]
```diff
@@ -601,7 +601,7 @@ void Analyzer::RemoveSupportAnalyzer(SupportAnalyzer* analyzer)
 
 	// We mark the analyzer as being removed here, which will prevent it
 	// from being used further. However, we don't actually delete it
-	// before the parent gets destroyed. While we woulc do that, it's a
+	// before the parent gets destroyed. While we would do that, it's a
```
Comment (jsoref) [inline src/analyzer/Analyzer.cc:604]: could?

File: src/analyzer/protocol/smb/smb1-protocol.pac [hunk 13]
```diff
@@ -135,7 +135,7 @@ type SMB_Message(header: SMB_Header, offset: uint16, command: uint8, is_orig: bo
 };
 
 type SMB_andx_command(header: SMB_Header, is_orig: bool, offset: uint16, command: uint8) = case command of {
-	0xff    -> no_futher_commands : empty;
+	0xff    -> no_further_commands : empty;
```
Comment (jsoref) [inline src/analyzer/protocol/smb/smb1-protocol.pac:138]: i could adjust the whitespace...

File: src/file_analysis/analyzer/hash/Hash.h [hunk 14]
```diff
@@ -34,7 +34,7 @@ class Hash : public file_analysis::Analyzer
 
 	/**
 	 * Finalizes the hash and raises a "file_hash" event.
-	 * @return always false so analyze will be deteched from file.
+	 * @return always false so analyze will be detached from file.
```
Comment (jsoref) [inline src/file_analysis/analyzer/hash/Hash.h:37]: ?

File: src/packet_analysis/protocol/ip/SessionAdapter.h [hunk 15]
```diff
@@ -90,7 +90,7 @@ class SessionAdapter : public analyzer::Analyzer
 	/**
 	 * Helper to raise a \c packet_contents event.
 	 *
-	 * @param data The dass to pass to the event.
+	 * @param data The data to pass to the event.
```
Comment (jsoref) [inline src/packet_analysis/protocol/ip/SessionAdapter.h:93]: ?

File: src/parse.y [hunk 16]
```diff
@@ -1051,7 +1051,7 @@ enum_body_list:
 enum_body_elem:
 		/* TODO: We could also define this as TOK_ID '=' expr, (or
 		   TOK_ID '=' = TOK_ID) so that we can return more descriptive
-		   error messages if someboy tries to use constant variables as
+		   error messages if somebody tries to use constant variables as
```
Comment (jsoref) [inline src/parse.y:1054]: ?

File: src/script_opt/CPP/Compile.h [hunk 17]
```diff
@@ -10,7 +10,7 @@
 #include "zeek/script_opt/ScriptOpt.h"
 
 // We structure the compiler for generating C++ versions of Zeek script
-// bodies maily as a single large class.  While we divide the compiler's
+// bodies mainly as a single large class.  While we divide the compiler's
```
Comment (jsoref) [inline src/script_opt/CPP/Compile.h:13]: ?

File: testing/btest/language/enum-nonexisting.zeek [hunk 18]
```diff
@@ -1,6 +1,6 @@
 # @TEST-EXEC-FAIL: zeek -b %INPUT  >output 2>&1
 # @TEST-EXEC: TEST_DIFF_CANONIFIER="$SCRIPTS/diff-remove-abspath" btest-diff output
 
-redef enum notexisting += {
+redef enum nonexistent += {
```
Comment (jsoref) [inline testing/btest/language/enum-nonexisting.zeek:4]: not sure if this change is ok

File: testing/btest/supervisor/config-bare-mode.zeek [hunk 19]
```diff
@@ -34,7 +34,7 @@ event zeek_init()
 		Broker::subscribe(topic);
 		Broker::listen("127.0.0.1", to_port(getenv("BROKER_PORT")));
 
-		# Create a node that inherits basre mode from us.
+		# Create a node that inherits base mode from us.
```
Comment (jsoref) [inline testing/btest/supervisor/config-bare-mode.zeek:37]: ?

**Review by jsoref — COMMENTED** [review]
Most corrections suggested by Google Sheets. All fault mine.

**Comment by timwoj** [comment]
Wow, that's a rather large PR. Thanks!

We're happy to take the changes to spelling for comments, but changes in code should probably be reverted. There's API guarantees for various files, and we can't guarantee that changing the name of a variable (even if it's spelled wrong) doesn't break external code somewhere.

**Comment by jsoref** [comment]
I've dropped a [handful of things which looked like they'd qualify as variables](https://github.com/zeek/zeek/compare/58a0de5d068a1288e4eac5d446aad07c834efcfc..d8d7723b4f54ccc7bd511f8ef57362428b23d5c3)

**Comment by jsoref** [comment]
Just realized that there was useful stuff in the check-run output, so [this should address most of the items](https://github.com/zeek/zeek/compare/d8d7723b4f54ccc7bd511f8ef57362428b23d5c3..249c47c3f6475a53203a577732676f573ce33e28)

**Comment by ckreibich** [comment]
I'm with Tim here — I'll just close this since we won't get to it in the near future anyway. @jsoref, if you could focus this on comments only (and let's say in the `src` and `scripts` folders, to start with), we can most likely revisit this. We'll never merge hundreds of commits into our project history, so at a minimum we'll squash this down to a handful.

> Fwiw, this is a medium size PR for me (I do 1k commit PRs often enough).

This statement should give you pause. No maintainer wants to have to work through a 1k commit PR.

**Pull request closed without merging**